src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/__tests__/DeltaOnboardingViewControllerTests.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+import XCTest
+@testable import ENA
+
+class DeltaOnboardingViewControllerTests: XCTestCase {
+	
+	override func setUp() {
+		super.setUp()
+	}
+	
+	private func createVC() -> DeltaOnboardingV15ViewController {
+		AppStoryboard.onboarding.initiate(viewControllerType: DeltaOnboardingV15ViewController.self) { coder -> UIViewController? in
+			DeltaOnboardingV15ViewController(
+				coder: coder,
+				supportedCountries: [ Country.defaultCountry() ]
+			)
+		}
+	}
+	
+	private func createVCWithoutCountries() -> DeltaOnboardingV15ViewController {
+		AppStoryboard.onboarding.initiate(viewControllerType: DeltaOnboardingV15ViewController.self) { coder -> UIViewController? in
+			DeltaOnboardingV15ViewController(
+				coder: coder,
+				supportedCountries: []
+			)
+		}
+	}
+	
+	func testCellsInSection0() {
+		let vc = createVC()
+		_ = vc.view
```

Comment:
```suggestion
		vc.loadViewIfNeeded()
``` 
is cleaner to read and understand. Following appearances also.

Reply:
changed